product_docs/docs/biganimal/release/getting_started/preparing_cloud_account/preparing_gcp/index.mdx

```diff
@@ -1,3 +1,76 @@
 ---
 title: Preparing your Google Cloud account 
----
\ No newline at end of file
+---
+
+BigAnimal requires you to check the readiness of your Google Cloud (GCP) account before you deploy your clusters. (You don't need to perform this check if you're using BigAnimal's cloud account as your [deployment option](/biganimal/release/overview/deployment_options).) The checks that you perform ensure that your Google Cloud account is prepared to meet your clusters' requirements and resource limits.
+
+## Required roles
+
+Ensure you have at least the following combined roles:
+- roles/iam.serviceAccountCreator
+- roles/iam.serviceAccountKeyAdmin
+- roles/iam.roleAdmin
+- roles/resourcemanager.projectIamAdmin
+- roles/compute.viewer
+
+Or an equivalent single role such as:
+- roles/owner
+
+
+## Check Google Cloud resource limits for running BigAnimal
+
+EDB provides a shell script, called [biganimal-csp-preflight](https://github.com/EnterpriseDB/cloud-utilities/blob/main/gcp/biganimal-csp-preflight), which checks whether requirements and resource limits are met in your Google Cloud account based on the clusters you plan to deploy.
+
+1.  Open [Google Cloud Shell](https://cloud.google.com/shell) in your browser.
+2.  From Google Cloud Shell, run the following command:
+
+    ```shell
+    curl -sL https://raw.githubusercontent.com/EnterpriseDB/cloud-utilities/main/gcp/biganimal-csp-preflight | bash -s <GCP-account-ID> <region> [options]
+    ```
+    The required arguments are:
+
+    | Argument | Description |
+    | -------- | ----------- |
+    | &lt;account-id> | Google Cloud account ID of your BigAnimal deployment. |
```

Comment:
```suggestion
    | <account-id> | Google Cloud account ID of your BigAnimal deployment. |
```

Reply:
We need to use the code for the left angle brackets or we get an error when generating the output